_data/events.json

```diff
@@ -0,0 +1,321 @@
+      "pastTenseOpeningParagraph": "The <strong>Fourth AmericasNLP Competition</strong> took place online in June, at NAACL 2024 in Mexico City. The competition focused on creating machine translation systems for indigenous languages from the Americas.",
+      "links": [
+         "<a href='http://turing.iimas.unam.mx/americasnlp/2024_workshop.html'>turing.iimas.unam.mx/americasnlp/2024_workshop.html</a>"
+      ],
+      "sharedTasks": [
+         "A machine translation shared task on truly low-resource languages.",
+         "A shared task on morphological adaptation to generate educational examples."
+      ],
+      "speakers": [
+         {
+            "type": "Keynote speakers",
+            "about": [
+               "Graham Neubig",
+               "Jaime Pérez González"
+            ]
+         },
+         {
+            "type": "Organisers",
+            "about": [
+               "Manuel Mager",
+               "Abteen Ebrahimi",
+               "Shruti Rijhwani",
+               "Arturo Oncevay",
+               "Luis Chiruzzo",
+               "Robert Pugh",
+               "Katharina Kann"
+            ]
+         }
+      ],
+      "importantDates": [
+         {
+            "name": "Start of the anonymity period",
+            "date": "10 February"
+         },
+         {
+            "name": "Submission deadline",
+            "date": "10 March"
+         },
+         {
+            "name": "Notification of acceptance",
+            "date": "24 April"
+         },
+         {
+            "name": "Camera ready papers",
+            "date": "24 April"
+         },
+         {
+            "name": "Workshop",
+            "date": "20 or 21 June"
+         }
+      ]
+   },
+   {
+      "name":"IWSLT 2024",
+      "id":"iwslt-2024",
+      "description":"International Conference on Spoken Language Translation (IWSLT)",
+      "callsForPapersDeadline":"2024-04-29",
+      "startDate":"2024-08-15",
+      "endDate":"2024-08-16",
+      "location":{
+         "name":"Bangkok, Thailand",
+         "type":"Place",
+         "eventAttendanceMode":"MixedEventAttendanceMode"
+      },
+      "organizer":{
+         "type":"Person",
+         "url":"https://iwslt.org/2023/#organizers"
+      },
+      "futureTenseOpeningParagraph":"The International Conference on Spoken Language Translation (<strong>IWSLT</strong>) is an annual scientific conference on spoken language translation.<br> <br> IWSLT 2024 will be co-located at ACL 2024 online and in Bangkok, Thailand from 15 to 16 August, 2024. <br><br> IWSLT 2024 will be a hybrid event.",
+      "pastTenseOpeningParagraph":"The nineteenth Machine Translation Summit 2023 (<strong>MTS 2023</strong>) took place from 4 September to 8 September in Macau Special Administrative Region, China. MTS 2023 was organised by [AAMT](/aamt).",
```

Comment:
these 2 paragraphs are not the same, the past one has a different year and mixes in Markdown.